specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.6-preview.2/keys.json

```diff
@@ -1538,7 +1538,7 @@
     "KeyAttestation": {
       "type": "object",
       "properties": {
-        "certificates": {
+        "certificateBundle": {
           "type": "string",
           "format": "base64url",
           "description": "A base64url-encoded string containing certificates in PEM format, used for attestation validation."
```

Comment:
I don't think this captures the feedback. "certificateBundle" is no more descriptive than "certificates". The concern here was that, unlike other APIs you already have, you're base64-encoding a PEM file that is already ASCII and contains base64-encoded certificates. The feedback was also in regard to the order, which I explained in chat afterward was not specified as guaranteed but many customers take a reliance on anyway.

The only thing different here I see is that you only purport to support PEM files as opposed to the other "certificates" fields/parameters you have, which are either base64url-encoded PKCS12 (PFX) files, or as-is PEM files.

So I don't think the name should change here or, if it does, I remember the feedback was more to call out these are PEM files e.g., `"certificatePemFile"` or something to that effect.